README.md

```diff
@@ -50,6 +50,28 @@ It is available for download while running the service at `http://localhost:port
 gets generated from code (specifically from Spring annotations) 
 
 ## Authentication Providers
+### Enabling and Selecting Authentication Providers
+The Login Service allows users to select which authentication providers they would like to use
+as well as the order in which the authentication methods get prioritized.
+
+This is done by setting a number (Greater than 0) on the `order` tag for each auth method.
+the number used indicates the order in which you would like to use the method.
+
+For Example:
+The 2 methods currently enabled are config specified users:
+```
+loginsvc.rest.auth.provider.users.order = 1
+```
+and ldap:
+```
+loginsvc.rest.auth.provider.ldap.order = 2
+```
+In the above example, both methods are enabled with the config specified users taking priority over Ldap.
+
+In order to disable an authentication protocol, set the `order` property to `0`
```

Comment:
I am unsure if this is a good idea or not (thinking about an `enabled` field instead), but probably we can use it like this, too.

Reply:
My Worry is giving the user too many properties to fiddle with to get this working.

So thought it would be a good idea to collapse the functionality into one property.

Reply:
Why is removing the (sub)config of a given provider not enough? Do we need two options to have some method disabled?

Reply:
I think for the sake of temporarily disabling methods, whether for debugging or if the auth endpoint goes down for some reason for example,, it's nice to be able to disable it without having to remove everything from the config. Otherwise the user would have to cut and paste the config or comment everything out so they don't lose that information.